src/Sdk/WebApi/WebApi/RawConnection.cs

```diff
@@ -0,0 +1,207 @@
+using System;
```

Comment:
most of this is `VssConnection` stuff, but stripped out vssf qualities from it. This is only used for new flows, so no risk for existing calls